code/_onclick/item_attack.dm

```diff
@@ -221,17 +221,14 @@
 	else
 		switch(I.damtype)
 			if("brute")
-				if(istype(src, /mob/living/carbon/slime))
-					M.adjustBrainLoss(power)
-				else
-					if(istype(M, /mob/living/carbon/monkey))
-						var/mob/living/carbon/monkey/K = M
-						power = K.defense(power,def_zone)
-					M.take_organ_damage(power)
-					if (prob(33) && I.force) // Added blood for whacking non-humans too
-						var/turf/location = M.loc
-						if (istype(location, /turf/simulated))
-							location:add_blood_floor(M)
+				if(istype(M, /mob/living/carbon/monkey))
+					var/mob/living/carbon/monkey/K = M
+					power = K.defense(power,def_zone)
+				M.take_organ_damage(power)
+				if (prob(33) && I.force) // Added blood for whacking non-humans too
+					var/turf/location = M.loc
+					if (istype(location, /turf/simulated))
+						location:add_blood_floor(M)
```

Comment:
: is fucked up and evil; use `location.add_blood_floor(M)` instead

Reply:
I know you didn't code this but anything updated should be improved wherever possible

Reply:
It'd be unatomic but sure.